src/edge-functions/inject-script.ts

```diff
@@ -63,5 +63,6 @@ export default async function handler(_request: Request, context: Context) {
 
 export const config: Config = {
   path: "/*",
+  cache: "manual",
```

Comment:
for all edge function defined routes responses themselves include `cache-control` headers, since we forward these headers the edge function results will be cached based on these headers

Reply:
Sounds good.